TestingApp/ViewController/ViewController+AuthTimer.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+extension ViewController {
+    /// We don't have a way to report back to viewController,
```

Comment:
Interesting, but then integrators also would need to do something like this?

Reply:
No, because it covers a case that should never happen. For us, it is just convenient to see that `refresh` button is back to being disabled and "deauthenticate" button label is changed to "authenticate". In reality, there won't be such buttons present in our clients' apps. We communicate auth issue to the visitor who needs to restart the app.